### DISCUSSION TITLE
Request for Switch homebrew build

### DISCUSSION THREAD

**Discussion by erfansvsh** [azahar-emu/azahar#545] [Ideas]
Hi there dear developer of Azahar,

I’m a Nintendo Switch user and have been testing Lime3DS. So far, its performance is much better than Citra. However, I had to install Android and Linux on my Switch to use Lime3DS, whereas Citra has a RetroArch libretro core that runs directly on the Switch through RetroArch or Lakka.

Given the overhead of Android and Linux, which negatively affects battery life, I attempted to create Switch builds from your Lime3DS GitHub project source code. You can see my 59 attempts here:

https://github.com/erfansvsh/lime3ds.github.io/actions/

Despite my efforts, I haven’t been successful due to limited resources.

Let’s consider a great example from the emulation community: “Flycast.” This project, a fork of “reicast,” significantly improved Dreamcast emulation. Flycast provides binaries for almost all platforms, including the Nintendo Switch, making it easy for users to build their own versions.

Check out their GitHub page:

https://flyinghead.github.io/flycast-builds/

I converted one of these Switch builds into a “.nsp” file and installed it as a game. Now, I have “Flycast” on my Switch as a game, running Dreamcast games excellently.

If Azahar had a similar setup, it would be even more impactful. Many Nintendo 3DS fans now own a Switch, and Azahar is a 3DS emulator after all.

I’m requesting a Switch homebrew build of Azahar. After many failed attempts to build it myself, I hope you can create and release it.

Running games on Horizon (the Switch’s default OS) optimizes power usage, extending battery life compared to other platforms. Additionally, many users lack the knowledge or desire to install Linux and Android on their Switch due to the space required.

The success of Flycast’s Switch “.nro” homebrew build shows the potential for Azahar. It allows players to enjoy Dreamcast emulation on the Switch easily.

Please consider building and releasing a Switch version of Azahar.

Thank you.

**Comment by OpenSauce04** [comment]
We do not have any plans to create a native build for the Switch.
